sampleconfig/core.yaml

```diff
@@ -678,6 +678,16 @@ ledger:
     # the minimum duration (in milliseconds) between writing
     # two consecutive db batches for converting the ineligible missing data entries to eligible missing data entries
     collElgProcDbBatchesInterval: 1000
+    # The missing data entries are classified into two categories:
+    # (1) prioritized
+    # (2) deprioritized
+    # Initially, all missing data are in the prioritized list. When the
+    # reconciler is unable to fetch the missing data from other peers,
+    # the unreconciled missing data would be moved to the deprioritized list.
+    # The reconciler would retry deprioritized missing data after every
+    # deprioritizedDataReconcilerInterval (unit: minutes). Note that the
+    # interval needs to be greater than the reconcileSleepInterval
+    deprioritizedDataReconcilerInterval: 60m
```

Comment:
I would have preferred to keep this config close to `reconcileSleepInterval` for usability and we could have a check in the code and print a warning if this is relatively smaller than that. However, that would criss-cross in the code unless we make reconciler control this and pass explicitly the category in the call... Per the current code, I am not sure what else can we do, so we can keep this as is.